src/backend/distributed/executor/multi_utility.c

```diff
@@ -2142,8 +2203,15 @@ ErrorIfDistributedAlterSeqOwnedBy(AlterSeqStmt *alterSeqStmt)
 		return;
 	}
 
-	/* see whether the sequences is already owned by a distributed table */
-	if (sequenceIsOwned(sequenceId, &ownedByTableId, &ownedByColumnId))
+#if (PG_VERSION_NUM >= 100000)
+	sequenceOwned = sequenceIsOwned(sequenceId, DEPENDENCY_AUTO, &ownedByTableId,
```

Comment:
I think it'd be better if we also checked for INTERNAL sequences (used by identity columns) here, for some future proofing.  For that matter, have you done any checking how identity columns work with citus and/or other v10 features?